packages/docusaurus-theme-classic/src/theme/hooks/useTheme.js

```diff
@@ -61,6 +61,14 @@ const useTheme = () => {
     }
   }, [setTheme]);
 
+  useEffect(() => {
+    window
```

Comment:
Do we need to check for `disableDarkMode` here?

Reply:
Done!